src/core/model.js

```diff
@@ -239,15 +223,15 @@ export default class Model extends Object3D {
     return Object.assign({}, uniforms, cameraUniforms, viewUniforms);
   }
 
-  /*
-   * @param {Camera} opt.camera=
-   * @param {Camera} opt.viewMatrix=
-   */
-  /* eslint-disable max-statements */
-  render(uniforms = {}) {
+  draw({uniforms = {}, attributes = {}, settings = {}, samplers = {}}) {
```

Comment:
> Is the draw() function here for compatibility reasons? I see that it just calls Model.render() with all arguments passed over.

Good question!

Actually, `draw` is new. The old function (`render`) only takes uniforms. As you can see, we want to be able to supply more parameters.  But everyone already uses `render` so if we change that function's signature, we will break a lot of code. 

So one way is simply to define a new function, `draw` and maintain both in parallel. But that's only one way. We could also do dynamic checking on the `render` arguments.

I will create an API-AUDIT doc and add bullets about API design issues that need to be settled and list this question here.